Sources/StreamChatUI/CommonViews/ContainerStackView.swift

```diff
@@ -289,7 +290,10 @@ public class ContainerStackView: UIView {
 
         Animate(isAnimated: animated) {
             subview.alpha = 0
-            self.hideConstraintsByView[subview]?.width.isActive = true
+
+            if self.axis == .horizontal {
+                self.hideConstraintsByView[subview]?.width.isActive = true
+            }
             self.hideConstraintsByView[subview]?.height.isActive = true
```

Comment:
Isn't there a missing `else` there?

Reply:
No, it is fine like this, only if horizontal we want to activate it, but if not, we don't need to anything more 👍